packages/api/endpoints/granules.js

```diff
@@ -140,37 +142,51 @@ async function del(req, res) {
   log.info(`granules.del ${granuleId}`);
 
   const granuleModelClient = new models.Granule();
+  const granulePgModel = new GranulePgModel();
 
-  let granule;
+  const knex = await getKnexClient({ env: process.env });
+
+  let dynamoGranule;
+  let pgGranule;
+
+  // If the granule does not exist in Dynamo, throw an error
   try {
-    granule = await granuleModelClient.getRecord({ granuleId });
+    dynamoGranule = await granuleModelClient.getRecord({ granuleId });
   } catch (error) {
     if (error instanceof RecordDoesNotExist) {
       return res.boom.notFound(error);
     }
     throw error;
   }
 
-  if (granule.detail) {
-    return res.boom.badRequest(granule);
-  }
-
+  // If the granule does not exist in PG, just log it
   try {
-    await granuleModelClient.delete(granule);
+    pgGranule = await granulePgModel.get(knex, { granule_id: granuleId });
   } catch (error) {
-    if (error instanceof DeletePublishedGranule) {
-      return res.boom.badRequest(error.message);
+    if (error instanceof RecordDoesNotExist) {
+      log.info(`Postgres Granule with ID ${granuleId} does not exist`);
+    } else {
+      throw error;
```

Comment:
I'd like to test this somehow: just the condition that if the error is not `RecordDoesNotExist`, the API returns an error code. But it seems pretty tricky to do since injecting mocks into API route handlers is tricky. So we can skip it

Reply:
Added a test here. I think we can mock `GranulePgModel.get()` to trigger this error:

https://github.com/nasa/cumulus/pull/2057/commits/667e41a603d23b4fc2f553be40504132806f95d0